erpnext/controllers/website_list_for_contact.py

```diff
@@ -28,7 +28,7 @@ def get_transaction_list(doctype, txt=None, filters=None, limit_start=0, limit_p
 	if doctype == 'Supplier Quotation':
 		filters.append((doctype, "docstatus", "<", 2))
 	else:
-		filters.append((doctype, "docstatus", "=", 1))
+		filters.append((doctype, "docstatus", "<", 2))
```

Comment:
use condition like `doctype in ['Supplier Quotation', 'Quotation', 'Sales Order']` then docstatus < 2 otherwise docstatus = 1 otherwise you'll affect more documents that you're expecting